src/main/resources/resources/internal_de.properties

```diff
@@ -0,0 +1,181 @@
+﻿
```

Comment:
Please rewrite the whole file matching the keys from the new base, both in definition (for example `label.login.title` in place of `label.login_title`) and in line numbering, with same keys on the same line.